demo/deploy-bookstore.sh

```diff
@@ -10,6 +10,7 @@ SVC="bookstore-$VERSION"
 kubectl delete deployment "$SVC" -n "$BOOKSTORE_NAMESPACE"  --ignore-not-found
 
 # Create a top level service just for the bookstore domain
+if [ "$ROUTES_V2" = "false" ]; then
```

Comment:
This condition is a work around for now and will be removed once we have traffic split implemented for routes v2.